### PR TITLE
[DRAFT] BUGFIX: fix xades test functionality

### PR DESCRIPTION
**Pull request by BanalitoRaulito**

### CODE REVIEW

File: app/Http/Controllers/SignLocallyController.php [hunk 1]
```diff
@@ -198,6 +198,7 @@ public function startSignCustomFile(Request $request)
             Session::put("issandbox-$fileId", true);
             return redirect()->to(url('/show-download-signed-file') . "?file_id=$fileId");
         } else {
+            Cache::put("prepared-files-$docId", $sourceFilesCache ?? null);
```
Comment (smartman) [inline app/Http/Controllers/SignLocallyController.php:201]: sourceFilesCache is undefined here and breaking the app

Comment (BanalitoRaulito) [reply app/Http/Controllers/SignLocallyController.php]: with nullish coalescin the sourceFilesCache can not be undefined

Comment (BanalitoRaulito) [reply app/Http/Controllers/SignLocallyController.php]: I updated it and set $sourceFilesCache to null for better readability.

**Comment by omitobi** [comment]
Why is this not merged?

**Comment by BanalitoRaulito** [comment]
closed because PR is outdated

**Pull request closed without merging**